Slim/Plugin/ExtendedBrowseModes/strings.txt

```diff
@@ -189,29 +202,34 @@ PLUGIN_EXTENDED_BROWSEMODES_DELETED
 PLUGIN_EXTENDED_BROWSEMODES_AUDIOBOOKS
 	DE	Hörbücher
 	EN	Audiobooks
+	FR	Livres audio
 	SV	Ljudböcker
 	NL	Luisterboeken
 
 PLUGIN_EXTENDED_BROWSEMODES_NO_AUDIOBOOKS
 	DE	Alles ohne Hörbücher
 	EN	All Music without Audiobooks
+	FR	Toute la musique sans les livres audio
 	SV	All musik förutom ljudböcker
 	NL	Alle muziek zonder luisterboeken
 
 PLUGIN_EXTENDED_BROWSEMODES_AUDIOBOOKS_DESC
 	DE	Erstelle eine "Hörbücher" und eine "Alles ohne Hörbücher" Ansicht, basierend auf den folgenden Stilrichtungen:
 	EN	Create an Audiobooks library view and a view without audio books based on the following genres:
+	FR	Créer une bibliothèque thématique de livres audio et une bibliothèque sans les livres audio en fonction des genres suivants :
 	SV	Skapa ett "delbibliotek för ljudböcker" och ett "Allt utom ljudböcker delbibliotek" baserat på följande genrer:
 	NL	Maak een luisterboek weergave en een weergave zonder luisterboeken gebaseerd op de volgende genres:
 
 PLUGIN_EXTENDED_BROWSEMODES_AUDIOBOOK_GENRES
 	DE	Hörbuch, Hörspiel, Comedy, Cabaret, Humor, Hörbuch & Gesprochene Inhalte, Hörbuch & gesprochene Inhalte, Kabarett, Podcast, Sprache, Audiobook, Audiobooks, Spoken, Speech
 	EN	Audiobook, Audiobooks, Spoken, Speech, Comedy, Spoken & Audio, Books & Spoken, Cabaret, Humour, Podcast
+	FR	Audiobook, Audiobooks, Spoken, Speech, Comedy, Spoken & Audio, Books & Spoken, Cabaret, Humour, Podcast, Livre audio, Livres audio, Parlé, Discours
```

Comment:
Feel free to add terms you know about. This doesn't need to be a literal translation.

Reply:
Thank you for the info. That's how I understood this genre list.

As the initial list seemed rather relevant to me (I never listen to audiobooks), I just added a few translations to the genres already present.
I left the words in English, because I figured someone might have downloaded / purchased audiobooks with these tags pre-positioned. I told myself that, in this case, it was better to have more proposals than not enough.